fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -106,9 +106,9 @@ private void exportBinary(final URI uri)
             writeResponse(response, file);
         }
     }
-    private void exportContainer(final URI uri)
+    private void exportDescription(final URI uri, final URI... describes)
             throws FcrepoOperationFailedException, IOException {
-        final File file = fileForContainer(uri);
+        final File file = fileForContainer(describes.length > 0 ? describes[0] : uri);
```

Comment:
fhmm, if we drop descriptions, that seems problematic. As @mikedurbin [mentioned](https://wiki.duraspace.org/display/FF/Export+Format?focusedCommentId=77449413#comment-77449413), the `fcr:metatadata` syntax may not work on all platforms. 
Would an indexed form make sense:
`.../rest/file1.ttl`, `.../rest/file1-1.ttl`, `.../rest/file1-2.ttl`, etc? Or some other naming convention?